### DISCUSSION TITLE
Asset paths not as expected

### DISCUSSION THREAD

**Discussion by Synchro** [filamentphp/filament#5257] [Help]
I have a pretty vanilla Filament 2 installation on Laravel 8, PHP 8, assets built with mix. I gather from the docs that Filament's css and js assets should be bundled into the host Laravel app's assets and served from `/public/js/app.js` and `/public/css/app.css` along with all the other app assets after they are built with mix (Filament's paths are added to tailwind's config). However, I'm finding it tries to load them from `/filament/assets/app.js?id=[...]` and `/filament/assets/app.css?id=[...]` instead, and those files don't exist, so it all falls to bits.

Is this wrong? Are these files supposed to exist?

**Comment by danharrin** [comment]
For the admin panel - all assets are precompiled and served from `/filament/assets` - which is inside our `/vendor` directory. If they 404, your web server is probably configured wrong. Default Valet and Forge should work fine - https://github.com/livewire/livewire/issues/242#issuecomment-948054946

For outside the admin panel - this is your own app, so you compile the assets yourself. You can use Mix or Vite. But I think you are using the admin panel, right?
